### PR TITLE
Portfolio - mobile menu branch

### PR DESCRIPTION
**Pull request by FredsephT32**
- Implemented the required mobile features.
- Properly used JavaScript syntax.
- Use JavaScript to manipulate DOM elements.
- we make sure that there are no linter errors.
- we make sure that you used the correct GitHub flow.
- we make sure that you documented your work in a professional way.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,12 +15,21 @@
 
         <nav class="nav-bar">
           <a href="#" class="Left-Logo">My Logo</a>
-          <a> <img src="Images/Union.png" id="Hamburger-logo" alt="Three-Lines"> </a>
+          <a> <img src="Images/Union.png" id="Hamburger-icon" alt="Three-Lines"> </a>
```
Comment (uwadonat) [inline index.html:18]: - [x] When the user clicks (or taps) the hamburger button, the mobile menu should appears. For now nothing happens. kindly fix this.
ha

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @FredsephT32,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your pull request message is very descriptive ✔️
- Well organized and clean code ✔️
- Professional readme file ✔️
almost
## Required Changes ♻️


_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by FredsephT32** [comment]
I don't understand the reviewer [uwadonat](https://github.com/uwadonat) because I double check my "mobile-menu" branch and when I click the hamburger button it works perfectly. 

Probably  [uwadonat](https://github.com/uwadonat)  did not review my last commit.

**Review by Reem-lab — APPROVED** [review]
# Hi @FredsephT32,👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉🔥🔥💯💯



## Highlights ✨✨
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the functionality✅
📌 Good commit messages ✅
📌 Professional README ✅

## Optional suggestions
- [ OPTIONAL ] I think it would be great if you blur the background since it is blured in the Figma template so please blur it and remove the scroll in the side since it is not found in the Figma as well 🔥🚀💯😊

- Your design
image

- Original design
image


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by FredsephT32** [comment]
Thank you [Reem-lab](https://github.com/Reem-lab) for approving my pull request!

**Pull request merged**